### ISSUE TITLE
Buffer.writeBigUInt64LE does not exist (but does in nodejs)

### ISSUE THREAD

**Issue by yohan1234** [electron/electron#19184]
Buffer.writeBigUInt64LE was added in nodejs 12.0:
https://nodejs.org/docs/v12.0.0/api/buffer.html#buffer_buf_writebiguint64le_value_offset

The following works in nodejs 12.6.0 but in the main process of Electron 5.0.6 it crashes with function not found:
```
const testBuf = Buffer.allocUnsafe(8);
testBuf.writeBigUInt64LE(BigInt("125125125125"), 0);
console.log(testBuf)
```
Expected printout:
`<Buffer 05 e4 09 22 1d 00 00 00>`

**Comment by DeerMichel** [comment]
Apparently, our node fork is missing 2 commits on the v12.x branch ([upstream](https://github.com/nodejs/node/commits/v12.x/lib/internal/buffer.js) vs. [fork](https://github.com/electron/node/commits/electron-node-v12.x/lib/internal/buffer.js)). Is there a specific reason for that or can I just cp them? I see it's implemented in the wip v12.4 branch.

cc @MarshallOfSound @nornagon

**Comment by MarshallOfSound** [comment]
Our `12.x` branch should have been updated with the exact state of `12.0.0`.  I'd ask @codebytere and @nitsakh

**Comment by DeerMichel** [comment]
So how should we deal with this? I cp'ed the 2 commits locally and had no problems.

**Comment by nornagon** [comment]
how come our 12.x branch isn't based on the real 12.0.0?

**Comment by yohan1234** [comment]
I would recommend the team to rebase Electron on the proper 12.0.0 version of nodejs as a minor update

**Comment by yohan1234** [comment]
Hey guys, just bumping this! 

Will this be fixed in Electron 7? If so, then I can close it.

**Comment by DeerMichel** [comment]
It's fixed from v6 onward since we bumped to Node 12.4 🎉

**Comment by yohan1234** [comment]
Great thanks!
